assemble/src/main/assemblies/component.xml

```diff
@@ -40,15 +40,48 @@
         <include>org.apache.commons:commons-math3</include>
         <include>org.apache.commons:commons-vfs2</include>
         <include>org.apache.thrift:libthrift</include>
+        <include>org.eclipse.jetty:jetty-continuation</include>
         <include>org.eclipse.jetty:jetty-http</include>
         <include>org.eclipse.jetty:jetty-io</include>
         <include>org.eclipse.jetty:jetty-security</include>
         <include>org.eclipse.jetty:jetty-server</include>
+        <!-- jetty-servlet only needed by old monitor -->
         <include>org.eclipse.jetty:jetty-servlet</include>
         <include>org.eclipse.jetty:jetty-util</include>
         <include>org.apache.htrace:htrace-core</include>
         <include>org.slf4j:slf4j-api</include>
         <include>org.slf4j:slf4j-log4j12</include>
+        <!-- Jersey/Jackson-based webservice -->
```

Comment:
> Jersey brings in a lot of dependencies that will have to be distributed with Accumulo

This is a good point and one that I had punted on originally. I agree with Jersey being ideal as the reference implementation for jax-rs; however, depending on the size of what we actually need to include to make it work, I could be swayed. I'm not sure what this actually looks like transitively.